config-linux.md

```diff
@@ -15,49 +15,3 @@ Valid values are the strings for capabilities defined in [the man page](http://m
         "CAP_NET_BIND_SERVICE"
     ]
 ```
-
-## User namespace mappings
-
-```json
-    "uidMappings": [
-        {
-            "hostID": 1000,
-            "containerID": 0,
-            "size": 10
-        }
-    ],
-    "gidMappings": [
-        {
-            "hostID": 1000,
-            "containerID": 0,
-            "size": 10
-        }
-    ]
-```
-
-uid/gid mappings describe the user namespace mappings from the host to the container.
-The mappings represent how the bundle `rootfs` expects the user namespace to be setup and the runtime SHOULD NOT modify the permissions on the rootfs to realize the mapping.
-*hostID* is the starting uid/gid on the host to be mapped to *containerID* which is the starting uid/gid in the container and *size* refers to the number of ids to be mapped.
-There is a limit of 5 mappings which is the Linux kernel hard limit.
-
-## Default Devices and File Systems
```

Comment:
On Mon, Oct 05, 2015 at 01:01:17PM -0700, Mrunal Patel wrote:
> > -## Default Devices and File Systems
> 
> This section could remain here as it is talking about the mounts and
> not the runtime details.

Currently devices are a purely runtime.json issue (although see [1],
where I suggest moving the mknod bits back into config.json).  So I'm
ok keeping the default mounts here, but the default devices seem like
they should be moved into the runtime.json docs.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/y_Fsa2_jJaM